src/schema/common.ts

```diff
@@ -142,11 +142,11 @@ export interface PageGenerator<
   ) => TReturn[];
 }
 
-export const getSearchQuery = (
-  param: string,
-): string => `SELECT to_tsquery('english',
-                                string_agg(lexeme || ':*', ' & ' order by positions)) AS query
-              FROM unnest(to_tsvector('english', process_text(${param})))`;
+export const getSearchQuery = (param: string): string =>
+  `SELECT to_tsquery('english', ${param}) AS query`;
```

Comment:
Did you check speed and hits on partials with this?
Ref: https://dailydotdev.slack.com/archives/C03C3JZPRU4/p1705389107962379?thread_ts=1705387675.301509&channel=C03C3JZPRU4&message_ts=1705389107.962379

Reply:
the production implementation does partials on all words, now i changed it to do only for the last word.
haven't checked speed because in your message you refer to `like`, here i use the same mechanism just changed a bit the query. I did check hits though.
I'm not 100% sure hence i want to try in production and worst case i'll revert

Reply:
I tried 3 methods there. Anyway happy to try, you asked me there if i recall to not go as it would slow down, so just wanted to check 🙏